rps.js

```diff
@@ -41,5 +32,18 @@ var compare = function(choice1, choice2){
   }
 }
 
-compare(userChoice, computerChoice);
-});
\ No newline at end of file
+$("#userChoice button").click(function(){
```

Comment:
Okay, let's break this down:

$ means it's a jquery function.

`#userChoice` refers do the div in the html with the id "userChoice". `button` is one of the buttons inside the `#userChoice` div.

`.click (function(){` means that the stuff inside activates when one of the buttons is clicked.